py/strato/common/log/coloringformatter.py

```diff
@@ -23,6 +27,10 @@ def format(self, record):
             record.startColor = self._GREEN
         elif record.levelno == _WARNING:
             record.startColor = self._YELLOW
+        elif record.levelno == _STEP:
+            record.startColor = self._LIGHT_GREEN
+            record.levelname = "STEP [%d]" % self.STEP_COUNT
+            self.STEP_COUNT += 1
```

Comment:
what will happen if you run with multiple threads? how the count will behave? I thinks you need a lock here